deny.toml

```diff
@@ -0,0 +1,268 @@
+# This template contains all of the possible sections and their default values
```

Comment:
This seems to be a copy-pasted template from somewhere. Please add a comment to the top that explains in human language what we're trying to accomplish here.

If I understand correctly there is a list of explicitly licenses, which includes MIT / Apache2 style licenses, and everything else is forbidden. Plus an exception for zlib. 

Perhaps this would be more readable if you removed all the comments and commented-out defaults from the template and only included the effective parts. Not sure.

Reply:
This file was generated by `cargo deny init`. The comment has been added to explain the reasoning.

Yes, there is a list of licenses allowed in our projects. I've checked all these types of licenses, and they allow free usage for the libraries.

Frankly saying, I can put the `Zlib` license under the standard list.

I removed all comments.